shared/panels/project-update-sidepanel/_components/project-lead/project-lead.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from "react";
+import { Controller, useFormContext } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+
+import { Badge } from "@/design-system/atoms/badge";
+import { Accordion } from "@/design-system/molecules/accordion";
+import { MenuItemAvatarPort } from "@/design-system/molecules/menu-item";
+
+import { GithubUserAutocomplete } from "@/shared/features/user/github-user-autocomplete/github-user-autocomplete";
+import { useAuthUser } from "@/shared/hooks/auth/use-auth-user";
+import { EditProjectFormData } from "@/shared/panels/project-update-sidepanel/project-update-sidepanel.types";
+
+import { ProjectLeadProps } from "./project-lead.types";
+
+export function ProjectLead({ project }: ProjectLeadProps) {
+  const { t } = useTranslation("panels");
+  const { user } = useAuthUser();
+  const [invitedUser, setInvitedUser] = useState<MenuItemAvatarPort<number>[]>([]);
+  const { control } = useFormContext<EditProjectFormData>();
+
+  function findUserInProjectLead(id: string) {
+    return project.leaders?.find(lead => lead.id === id);
+  }
+
+  function findUserInInvited(githubId: number) {
+    const usersItems = project.invitedLeaders.map(user => ({
+      id: user.githubUserId,
+      label: user.login,
+      searchValue: user.login,
+      avatar: { src: user.avatarUrl },
+    }));
+
+    return usersItems?.find(lead => lead.id === githubId) || invitedUser?.find(lead => lead.id === githubId);
+  }
```

Comment:
Move to core ?

Reply:
don't think this one is useful because it's link to a component and a state